sumologic/resource_sumologic_dashboard_test.go

```diff
@@ -0,0 +1,551 @@
+					resource.TestCheckResourceAttr("sumologic_dashboard.tf_crud_test",
+						"time_range.0.begin_bounded_time_range.0.from.0.literal_time_range.0.range_name",
+						newLiteralRangeName),
+					resource.TestCheckResourceAttr("sumologic_dashboard.tf_crud_test",
+						"panel.#", "2"),
+					resource.TestCheckResourceAttr("sumologic_dashboard.tf_crud_test",
+						"panel.0.key", textPanel.Key),
+					resource.TestCheckResourceAttr("sumologic_dashboard.tf_crud_test",
+						"panel.1.key", searchPanel.Key),
+					resource.TestCheckResourceAttr("sumologic_dashboard.tf_crud_test",
+						"panel.1.sumo_search_panel.0.description", searchPanel.Description),
+					resource.TestCheckResourceAttr("sumologic_dashboard.tf_crud_test",
+						"panel.1.sumo_search_panel.0.query.0.query_key", searchPanel.Queries[0].QueryKey),
+					resource.TestCheckResourceAttr("sumologic_dashboard.tf_crud_test",
+						"layout.0.grid.0.layout_structure.#", "2"),
+					resource.TestCheckResourceAttr("sumologic_dashboard.tf_crud_test",
+						"layout.0.grid.0.layout_structure.0.key", newLayout.LayoutStructures[0].Key),
+					resource.TestCheckResourceAttr("sumologic_dashboard.tf_crud_test",
+						"layout.0.grid.0.layout_structure.1.key", newLayout.LayoutStructures[1].Key),
+					resource.TestCheckResourceAttr("sumologic_dashboard.tf_crud_test",
+						"variable.#", "2"),
+					resource.TestCheckResourceAttr("sumologic_dashboard.tf_crud_test",
+						"variable.0.name", newVariables[0].Name),
+					resource.TestCheckResourceAttr("sumologic_dashboard.tf_crud_test",
+						"variable.0.source_definition.0.csv_variable_source_definition.0.values",
+						newVariables[0].SourceDefinition.(CsvVariableSourceDefinition).Values),
+					resource.TestCheckResourceAttr("sumologic_dashboard.tf_crud_test",
+						"variable.1.name", newVariables[1].Name),
+					resource.TestCheckResourceAttr("sumologic_dashboard.tf_crud_test",
+						"variable.1.source_definition.0.log_query_variable_source_definition.0.query",
+						newVariables[1].SourceDefinition.(LogQueryVariableSourceDefinition).Query),
+					resource.TestCheckResourceAttr("sumologic_dashboard.tf_crud_test",
+						"variable.1.source_definition.0.log_query_variable_source_definition.0.field",
+						newVariables[1].SourceDefinition.(LogQueryVariableSourceDefinition).Field),
+				),
+			},
+		},
+	})
+}
+
+func testAccCheckDashboardDestroy(dashboard Dashboard) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		client := testAccProvider.Meta().(*Client)
+		_, err := client.GetDashboard(dashboard.ID)
+		if err == nil {
+			return fmt.Errorf("Dashboard (id=%s) still exists", dashboard.ID)
+		}
+		// for _, r := range s.RootModule().Resources {
+		// 	id := r.Primary.ID
+		// 	u, err := client.GetDashboard(id)
+		// 	if err != nil {
+		// 		return fmt.Errorf("Encountered an error: " + err.Error())
+		// 	}
+		// 	if u != nil {
+		// 		return fmt.Errorf("Dashboard %s still exists", id)
+		// 	}
+		// }
```

Comment:
you can do something like this instead of commenting out
```
for _, rs := range s.RootModule().Resources {
	if rs.Type != "sumologic_dashboard" {
		continue
	}
```
but since we only create dashboard it's not even necessary

Reply:
I was planning to remove it but I forgot. Will remove it.